src/main/credentialServer/httpCredentialHelperServer.ts

```diff
@@ -211,4 +300,8 @@ export class HttpCredentialHelperServer {
 
     return stdout;
   }
+
+  protected ensureEndsWithNewline(s: string): string {
+    return /\n$/.test(s) ? s : `${ s }\n`;
+  }
```

Comment:
We already import this from `@/utils/string`.

Reply:
I dropped the import because it's not used elsewhere.